dirty_equals/_other.py

```diff
@@ -17,6 +17,12 @@
     from typing_extensions import Literal  # type: ignore[assignment]
 
 
+class _Version(NamedTuple):
```

Comment:
Seems like overkill to have a named tuple here, a simple `Tuple[int, ...]` should be sufficient.

Reply:
fixed! 😄